superset/utils/cache_manager.py

```diff
@@ -24,28 +24,24 @@ class CacheManager:
     def __init__(self) -> None:
         super().__init__()
 
-        self._tables_cache = None
-        self._cache = None
```

Comment:
It's interesting that the 

```python
@property
def cache(self) -> Cache:
    return self._cache
```

method didn't barf from a MyPy perspective given that `self._cache` is actually `Optional[Cache]`.